### PR TITLE
Update Revise-submissions.mdx

### PR DESCRIPTION
**Pull request by emmahodcroft**
Add a line about using only original metadata for revisions for now, or contact us!

Preview: https://preview-update-revision-docs.pathoplexus.org/docs/how-to/revise-submissions

### CODE REVIEW

**Review by anna-parker — APPROVED** [review]
Lgtm!

File: monorepo/website/src/pages/docs/how-to/revise-submissions.mdx [hunk 1]
```diff
@@ -17,6 +17,8 @@ Submitting these changes ("revisions") will cause the version of the sequence to
 The process of submitting revisions is very similar to original submission.
 The main difference is that you must provide an `accession` column in the metadata file, which contains the Pathoplexus accession number(s) assigned when the sequences were submitted originally.
 
+**You must use the _original metadata_ when doing a revision!** (In other words, the original files you submitted to Pathoplexus.) Currently you will need to have this file locally - we aim to introduce a way to download this from Pathoplexus soon. If you do not have these files, please _contact us_ to help you do the revision.
+
```
Comment (anna-parker) [inline monorepo/website/src/pages/docs/how-to/revise-submissions.mdx:21]: or wait should we give an email to contact us at?

Comment (emmahodcroft) [reply monorepo/website/src/pages/docs/how-to/revise-submissions.mdx:21]: Ah good point. I can add this!

File: monorepo/website/src/pages/docs/how-to/revise-submissions.mdx [hunk 2]
```diff
@@ -17,6 +17,8 @@ Submitting these changes ("revisions") will cause the version of the sequence to
 The process of submitting revisions is very similar to original submission.
 The main difference is that you must provide an `accession` column in the metadata file, which contains the Pathoplexus accession number(s) assigned when the sequences were submitted originally.
 
+**You must use the _original metadata_ when doing a revision!** (In other words, the original files you submitted to Pathoplexus.) Currently you will need to have this file locally - we aim to introduce a way to download this from Pathoplexus soon. If you do not have these files, please _contact us_ to help you do the revision.
```
Comment (emmahodcroft) [inline monorepo/website/src/pages/docs/how-to/revise-submissions.mdx:20]:
```suggestion
**You must use the _original metadata_ when doing a revision!** (In other words, the original files you submitted to Pathoplexus.) Currently you will need to have this file locally - we aim to introduce a way to download this from Pathoplexus soon. If you do not have this file, please _contact us_ at [revisions@pathoplexus.org](mailto:revisions@pathoplexus.org) to help you do the revision.

```

File: monorepo/website/src/pages/docs/how-to/revise-submissions.mdx (hunk 1 above)
Comment (emmahodcroft) [reply monorepo/website/src/pages/docs/how-to/revise-submissions.mdx:21]: I also fixed the going back and forth between plural and singular 'file(s)'. What do you think?

**Pull request merged**